test/helpers/browser-env.js

```diff
@@ -0,0 +1,22 @@
+import { DOMParser } from 'xmldom'
+import { jsdom } from 'jsdom'
+import { File } from 'file-api'
+
+
+global.document = jsdom('<body></body>')
+global.DOMParser = DOMParser
+
+File.prototype.slice = function (start = 0, end = null, contentType = '') {
```

Comment:
Since our `supported` method determines if a browser is supported or not, how do we test that given that this browser was designed to be compatble?